pkg/feature/feature.go

```diff
@@ -0,0 +1,46 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2019 Red Hat, Inc.
+ */
+
+package feature
+
+import (
+	"kubevirt.io/kubevirt-cpu-nfd-plugin/pkg/file"
+)
+
+var LibvirtCPUMapFolder = "/etc/kubernetes/node-feature-discovery/source.d/cpu_map/"
+
+//LoadFeatures loads features for given cpu name
+func LoadFeatures(cpuModelName string) (map[string]bool, error) {
+	cpuFeatures := FeatureModel{}
+	cpuFeaturepath := GetPathCPUFefatures(cpuModelName)
```

Comment:
Sorry, I'm missing who's populating the cpu_map/ subfolder. Could you please remind me the flow?

Reply:
cpu_map is populated from libvirt container during labeller initialization (I have to create another PR to https://github.com/ksimon1/kubevirt-node-labeller/blob/master/kubevirt-node-labeller.yaml#L83)